manager.py

```diff
@@ -224,10 +225,18 @@ def askResponse(player, action):
 			player.coin = 10
 
 		if claimedIdentity == "Witch":
+			moreCoinPlayers = []
+
 			if player.order == userOrder:
 				target = int(raw_input("Which player's fortune do you want to swap with? "))
 			else:
-				target = int(random.randint(0, playersNum))
+				# get the list of player who has more coin
+				for moreCoinPlayer in playerList:
+					if moreCoinPlayer.order != player.order:
+						if moreCoinPlayer.coin >= player.coin:
+							moreCoinPlayers.append(moreCoinPlayer)
+				targetPlayer = random.choice(moreCoinPlayers)
+				target = targetPlayer.order
```

Comment:
you have already had targetPlayer in line 238, so I don't think you need to have another extra variable to store the player order.